library/2.1.5/validations.py

```diff
@@ -135,6 +135,13 @@ def valid_fs_path_or_raise(path: str):
     return path
 
 
+def allowed_fs_host_path_or_raise(path: str):
+    disallowed_paths = ["/mnt"]
```

Comment:
Might make sense to disallow `/` as well.

Reply:
Also `/var/db` and `/data`

Reply:
Oh, and `/audit` :)

Reply:
`/` is already blocked in both container and host paths. (check above func)
Will add the rest as well